lib/kernel.ts

```diff
@@ -231,6 +233,53 @@ const patchContract = (
 	}, contract);
 };
 
+/**
+ * @summary Upsert a contract in an unsafe way (DANGEROUS)
+ * @function
+ * @public
+ *
+ * @description
+ * This bypasses the whole permission system, so use with care.
+ *
+ * This function has the added limitation that you can only insert
+ * contracts of types that are defined in the Jellyfish core.
+ *
+ * @param {Object} context - exectuion context
+ * @param {Object} backend - backend
+ * @param {Object} contract - contract
+ * @returns {Object} contract
+ *
+ * @example
+ * const contract = await unsafeUpsertContract(backend, {
+ *   type: 'foo',
+ *   links: {},
+ *   requires: [],
+ *   capabilities: [],
+ *   tags: [],
+ *   active: true,
+ *   data: {
+ *     foo: 'bar'
+ *   }
+ * })
+ *
+ * console.log(contract.id)
+ */
+export const unsafeUpsertCard = async (
```

Comment:
IMO it's much more intuitive to have this in the kernel rather than in the script dealing with resolving permissions (especially because this function is completely skipping authorization), but I agree that this is not in the scope of this PR.

Reply:
That makes sense.